src/components/HookForm/index.jsx

```diff
@@ -168,6 +168,12 @@ export default class HookForm extends Component {
       scheduleTextField: '',
       taskValidJson: true,
       triggerSchemaValidJson: true,
+      validation: {
```

Comment:
Please add `validation: {},` on the initial state object above.

Reply:
Do I have to add the empty object, or better to instantiate it with owner prop as seen in the comment below. Im asking this because both of the comments are refering to the same code.

Reply:
An empty object is good enough.